controller.py

```diff
@@ -194,6 +194,19 @@ def search(self, ip_address=None, live=True):
             )
             return Response(body, 200, headers)
 
+    def library(self, uuid):
+        library = Library.for_urn(self._db, "urn:uuid:" + uuid)
```

Comment:
The specific place I'd apply the change suggested in my review comment is here -- only add 'urn:uuid' to the incoming `uuid` if it's not already present.